projects/client/RabbitMQ.Client/src/client/impl/Connection.cs

```diff
@@ -1359,13 +1344,12 @@ void StartAndTune()
                 connectionTune.m_frameMax);
             FrameMax = frameMax;
 
-            var heartbeat = (ushort)NegotiatedMaxValue(m_factory.RequestedHeartbeat,
-                connectionTune.m_heartbeat);
-            Heartbeat = heartbeat;
+            TimeSpan requestedHeartbeat = m_factory.RequestedHeartbeat; // TODO LRB
```

Comment:
What is left to do here @lukebakken?

Reply:
Probably nothing, I'm wrapping this up today.